src/components/EventManagement/EventAttendance/AttendanceList/AttendedEventList.tsx

```
@@ -60,14 +60,14 @@ const AttendedEventList: React.FC<Partial<InterfaceEvent>> = ({ _id }) => {
         <TableBody className="bg-primary">
           {event && (
             <TableRow
-              key={event._id}
+              key={event.id ?? event._id}
               className="bg-white rounded"
               role="row"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<tr>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)